libr/core/anal_tp.c

```diff
@@ -73,17 +73,10 @@ static void var_rename(RAnal *anal, RAnalVar *v, const char *name, ut64 addr) {
 	r_anal_var_rename (anal, fcn->addr, 1, v->kind, v->name, name, false);
 }
 
-static void var_retype(RAnal *anal, RAnalVar *var, const char *vname, char *type, ut64 addr, bool ref, bool pfx) {
-	if (!type || !var) {
-		return;
-	}
-	char *trim = type;
-	r_str_trim (trim);
-	if (!*trim) {
-		return;
-	}
+static void var_retype(RAnal *anal, RAnalVar *var, const char *vname, const char *type, ut64 addr, bool ref, bool pfx) {
+	r_return_if_fail (anal && var && type);
```

Comment:
this is causing multiple warnings in travis.

Reply:
where